ppb/sprites.py

```diff
@@ -140,7 +140,34 @@ def _attribute_gate(self, attribute, bad_sides):
             raise AttributeError(message)
 
 
-class BaseSprite(EventMixin):
+class Rotatable:
+    """
+    A simple rotation mixin. Can be included with sprites.
+    """
+    _rotation = 0
+    # This is necessary to make facing do the thing while also being adjustable.
+    basis = Vector(0, -1)
+    # Considered making basis private, the only reason to do so is to
+    # discourage people from relying on it as data.
+
+    @property
+    def facing(self):
+        return Vector(*self.basis).rotate(self.rotation).normalize()
+
+    @property
+    def rotation(self):
+        return self._rotation
+
+    @rotation.setter
+    def rotation(self, value):
+        self._rotation = value % 360
```

Comment:
This can still be negative; should this use the same convention as `ppb-vector` and have `-180 < _rotation <= 180`?

Reply:
In the case of this interface, dunno if it matters. The tests have some negative rotation and it gives the equivalent positive rotation, which is a reasonable result, but if both you and @astronouth7303 feel like I should port that over here, I will.

Reply:
@pathunstrom I don't think it's much of an issue either way, I just wanted to check that was intentional.